config/base/crds/full/numaflow.numaproj.io_interstepbufferservices.yaml

```diff
@@ -489,6 +489,18 @@ spec:
                         type: string
                       resources:
                         properties:
+                          claims:
```

Comment:
what is this claims thingy?

Reply:
New optional property under `resource`.

```
	// Claims lists the names of resources, defined in spec.resourceClaims,
	// that are used by this container.
	//
	// This is an alpha field and requires enabling the
	// DynamicResourceAllocation feature gate.
	//
	// This field is immutable. It can only be set for containers.
	//
	// +listType=map
	// +listMapKey=name
	// +featureGate=DynamicResourceAllocation
	// +optional
```